addons/mail/models/mail_tracking_value.py

```diff
@@ -35,10 +34,21 @@ class MailTracking(models.Model):
 
     tracking_sequence = fields.Integer('Tracking field sequence', readonly=1, default=100, oldname='track_sequence')
 
+    @property
+    def groups(self):
+        # the attribute `groups` of `ìr.model.fields` is useless (empty table)
+        # => get groups throught the model
+        model = self.env[self.field.model]
+        return model._fields[self.field.name].groups
+
+
     @api.model
-    def create_tracking_values(self, initial_value, new_value, col_name, col_info, tracking_sequence):
+    def create_tracking_values(self, col_info, tracking_sequence):
```

Comment:
maybe comment what should be in col_info
At least put this in the commit message